src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -170,7 +170,11 @@ const useFetchChannelHook = () => {
           label: `saveChat-${message?.id}`,
           task: () => {
             return new Promise((resolve) => {
-              if (message?.type === 'deleted') {
+              if (
+                message?.type === 'deleted' ||
+                message?.message_type === 'notification-deleted' ||
+                message?.type === 'notification-deleted'
+              ) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition checks for `message?.type === 'deleted'` and `message?.type === 'notification-deleted'` are redundant as they are checked twice. You can simplify this by removing the duplicate conditions.
```diff
-               if (
-                 message?.type === 'deleted' ||
-                 message?.message_type === 'notification-deleted' ||
-                 message?.type === 'notification-deleted'
-               ) {
+               if (
+                 message?.type === 'deleted' ||
+                 message?.message_type === 'notification-deleted'
+               ) {
                 resolve(true);
                 return;
               }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->